index.html

```diff
@@ -148,6 +148,43 @@ <h1>Multi-Post Stories</h1>
  
             </article>
         </section>
+        <section class="contact-form">
+            <h3>CONTACT US</h3>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p> 
+            <form class="form" action="https://formspree.io/f/xoqrbwdq" method="POST">
+                <div>
+                    <label for="text">
+                        <input type="text" id="text" placeholder="Text input" required minlength="6" maxlength="30">
+                    </label>
+                </div>
+                <div>
+                    <label for="email">
+                        <input type="email" id="email" placeholder="yannickmugisha23@gmail.com" required>
+                    </label>
+                    
+                </div>
+                <div>
+                    <label for="textarea">
+                        <textarea name="textarea" cols="50" rows="8" placeholder="Enter text here" minlength="15" maxlength="500"></textarea>
+                    </label>
+                    
+                </div>
+                <div class="down-form">
+                    <button>See project</button>

```

Comment:
- [ ] Kindly add the pressed and hover effect for the button. I can see the hover effect in the CSS but its not working properly on the site.